src/runtime/debug/arc-debug-handler.ts

```diff
@@ -39,9 +40,11 @@ export class ArcDebugHandler {
 
       this.arcDevtoolsChannel = devtoolsChannel.forArc(arc);
 
-      // Message handles go here.
-      const arcPlannerInvoker = new ArcPlannerInvoker(arc, this.arcDevtoolsChannel);
-      const arcStoresFetcher = new ArcStoresFetcher(arc, this.arcDevtoolsChannel);
+      if (!!listenerClasses) { // undefined => false
+        const listeners = listenerClasses.map(l => {
```

Comment:
My concern was that the original constructed the objects into const variables, though it never used them either, so I thought this might have been to ensure that the garbage collector didn't get to them, because they should live as long as this ArcDebugHandler does. Their registration with the devtools channel should guarantee that, but to be safe I thought I would do the same thing the original code did. I will drop the braces, though, as you suggest.

Reply:
Hmm. Removing the braces caused a compile failure. It would appear that the syntax of map is a bit odd. So I've kept the braces.

Reply:
I think const variable was there just to stop linter from complaining about a constructor being invoked and not assigned to anything. But it doesn't matter.

Reply:
Thanks Piotr. I can't test that the connections are being made, because of my laptop workflow. (All the code is on the workstation, and I don't forward X over ssh, so I can't run a webserver that my laptop can see.) So for now I'm going to leave the code as is. I will put in a todo, though, and perhaps we can tweak it up in SF next week.